### ISSUE TITLE
3.2.6 Traceback

### ISSUE THREAD

**Issue by blaggacao** [Kozea/wdb#129]
```
...
odoo_1           |   File "/opt/odoo/vendor/odoo/cc/odoo/models.py", line 415, in _build_model
odoo_1           |     import wdb; wdb.set_trace()  # breakpoint 0c8862f7 //
odoo_1           |   File "/usr/local/lib/python3.5/dist-packages/wdb/__init__.py", line 1086, in set_trace
odoo_1           |     wdb = Wdb.get(server=server, port=port)
odoo_1           |   File "/usr/local/lib/python3.5/dist-packages/wdb/__init__.py", line 118, in get
odoo_1           |     Wdb.__init__(wdb, server, port, force_uuid)
odoo_1           |   File "/usr/local/lib/python3.5/dist-packages/wdb/__init__.py", line 169, in __init__
odoo_1           |     self.get_breakpoints()
odoo_1           |   File "/usr/local/lib/python3.5/dist-packages/wdb/__init__.py", line 268, in get_breakpoints
odoo_1           |     breaks = loads(breaks or '{}')
odoo_1           |   File "/usr/lib/python3.5/json/__init__.py", line 319, in loads
odoo_1           |     return _default_decoder.decode(s)
odoo_1           |   File "/usr/lib/python3.5/json/decoder.py", line 339, in decode
odoo_1           |     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
odoo_1           |   File "/usr/lib/python3.5/json/decoder.py", line 357, in raw_decode
odoo_1           |     raise JSONDecodeError("Expecting value", s, err.value) from None
odoo_1           | json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

the value offending `breaks == 'Quit'; True`

**Comment by arkhan** [comment]
I have the same issue.

**Comment by blaggacao** [comment]
For the record I couldn't infer quickly where `Quit` is emitted in the server...

**Comment by GabbasovDinar** [comment]
I have the same issue too

**Comment by s3h10r** [comment]
same here (using the latest pypi release) :-/ does anyone has a quick fix for this?
```python
  File "/home/s3h10r/development/polaroidme/cli/./contactsheet", line 207, in <module>
    wdb.set_trace()
  File "/home/s3h10r/development/polaroidme/tmp/venv_polaroidme/lib/python3.6/site-packages/wdb/__init__.py", line 1086, in set_trace
    wdb = Wdb.get(server=server, port=port)
  File "/home/s3h10r/development/polaroidme/tmp/venv_polaroidme/lib/python3.6/site-packages/wdb/__init__.py", line 118, in get
    Wdb.__init__(wdb, server, port, force_uuid)
  File "/home/s3h10r/development/polaroidme/tmp/venv_polaroidme/lib/python3.6/site-packages/wdb/__init__.py", line 169, in __init__
    self.get_breakpoints()
  File "/home/s3h10r/development/polaroidme/tmp/venv_polaroidme/lib/python3.6/site-packages/wdb/__init__.py", line 268, in get_breakpoints
    breaks = loads(breaks or '{}')
  File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

**Comment by trojikman** [comment]
> same here (using the latest pypi release) :-/ does anyone has a quick fix for this?
> 
> ```python
>   File "/home/s3h10r/development/polaroidme/cli/./contactsheet", line 207, in <module>
>     wdb.set_trace()
>   File "/home/s3h10r/development/polaroidme/tmp/venv_polaroidme/lib/python3.6/site-packages/wdb/__init__.py", line 1086, in set_trace
>     wdb = Wdb.get(server=server, port=port)
>   File "/home/s3h10r/development/polaroidme/tmp/venv_polaroidme/lib/python3.6/site-packages/wdb/__init__.py", line 118, in get
>     Wdb.__init__(wdb, server, port, force_uuid)
>   File "/home/s3h10r/development/polaroidme/tmp/venv_polaroidme/lib/python3.6/site-packages/wdb/__init__.py", line 169, in __init__
>     self.get_breakpoints()
>   File "/home/s3h10r/development/polaroidme/tmp/venv_polaroidme/lib/python3.6/site-packages/wdb/__init__.py", line 268, in get_breakpoints
>     breaks = loads(breaks or '{}')
>   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
>     return _default_decoder.decode(s)
>   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
>     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
>   File "/usr/lib/python3.6/json/decoder.py", line 357, in raw_decode
>     raise JSONDecodeError("Expecting value", s, err.value) from None
> json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
> ```

Use version 3.2.5 for the wdb.server

**Comment by jasonmhite** [comment]
Same on py3.6.8.

**Comment by paradoxxxzero** [comment]
Could you please try with 3.3.0?

**Comment by paradoxxxzero** [comment]
Fixed see #131
